Super fast layered maps with Topojson.

nader77 opened the pull request:
#118

amitaibu commented:
@nader77 is this still relevant / ready for review?

nader77 commented:
@amitaibu This particular post is ready, But it should be a part of a series of posts about `AngularJs` maps, I'll work on them again and i'll let you know.

amitaibu commented:
I personally don't have much interest in Angular anymore ;) - so maybe change the post a bit so it could a be a standalone, and not part of a series?

nader77 commented:
Yes sure.

nader77 commented:
<img width="590" alt="super_fast_layered_maps_with_topojson" src="https://cloud.githubusercontent.com/assets/7369740/14286744/6bc652e8-fb50-11e5-9471-809c4c9d4308.png">

nader77 commented:
@amitaibu 
* I changed the view of the blog post to just talk about the difference between `GeoJSON` and `TopoJSON` and not regard angularJs as an important part.
* I changed my bio. (Tell me if you don't like it, I'll come up with something else - I don't like the old one)
* Fixed the behat test.

Please review.

amitaibu commented on app/_posts/2016-04-07-topojson.md:
```diff
@@ -0,0 +1,73 @@
+So while we were creating this map for [IBLI - Index-Based livestock insurance](http://ibli.ilri.org/category/ibli/), We used [angular-leaflet-directive](https://github.com/tombatossals/angular-leaflet-directive)
+, We got a huge GeoJSON from the client and with the help of this directive we were able to divide the map into provinces and display lots of information on each province.
+ 
+<div class="thumbnail">
```
use the new `{% include thumbnail.html image_path="assets/images/posts/facebook-post/image1.jpg" caption="Amitai Burstein's Facebook Post" %}` format

amitaibu commented on app/_posts/2016-04-07-topojson.md:
```diff
@@ -0,0 +1,73 @@
+Having an interactive map on your website is quite easy with [Leaflet](http://leafletjs.com/), 
+A map with layers to show you the information upon hovering, clicking, etc..  on a city, province, country, any kind of information you want, Whether you are using [ELM](http://elm-lang.org/), [AngularJs](https://angularjs.org/) or any other library.
+
+<!-- more -->
```
move below image

amitaibu commented:
Some technical stuff -- https://github.com/Gizra/Gizra#technical-stuff-to-write-posts

amitaibu commented:
After my comments, please assign to me, I'll review

amitaibu commented:
@nader77 you missed the [thumb image](https://github.com/Gizra/Gizra#technical-stuff-to-write-posts)

nader77 commented:
@amitaibu My main picture didn't look good as a thumbnail so I didn't add it, I'll try to resize it.

amitaibu commented:
you can add a different one then.

nader77 commented:
Added a cropped image:
<img width="1110" alt="gizra_s_blog" src="https://cloud.githubusercontent.com/assets/7369740/14430732/090bab3c-0003-11e6-9df1-242d307cb1c7.png">

nader77 commented:
@amitaibu Ready for code review.

RachelBaram commented:
@nader77  Is this still a live PR or can it be closed?

nader77 commented:
@RachelBaram I think it's irrelevant anymore, closing it.
